### PR TITLE
Rocket example db pool without deriving Clone

### LINKED ISSUE #425: Clone is not implemented for `sea_orm::DatabaseConnection` in rocket_example
I tried the `rocket_example` and I'm getting an error in `pool.rs`.

https://github.com/SeaQL/sea-orm/blob/82bad4a376efb875ac32ab601ac2cd9cce191e3f/examples/rocket_example/src/pool.rs#L12

```
the trait bound `sea_orm::DatabaseConnection: std::clone::Clone` is not satisfied

the trait `std::clone::Clone` is not implemented for `sea_orm::DatabaseConnection`rustc(E0277)
pool.rs(12, 5): the trait `std::clone::Clone` is not implemented for `sea_orm::DatabaseConnection`
```

Removing `#[derive(Clone)]` from `SeaOrmPool` fixes the error, but is it intended to have the `Clone` trait?

Features I was using:
```
sea-orm = { version = "0.5.0", features = ["sqlx-postgres", "macros", "runtime-tokio-rustls"]}
sea-orm-rocket = { git = "https://github.com/SeaQL/sea-orm" }
```

### PR DESCRIPTION
**Pull request by billy1624**
## PR Info
- Resolve #425

### CODE REVIEW

**Comment by 05storm26** [comment]
Instead of this, can you make `MockDatabaseConnection` clonable? 

If we could do that, then by default (having the default features enabled) `sea_orm::DatabaseConnection` could be `Clone` by default. Some frameworks like actix_web require app global datas to be clonable. Currently one can only disable the mock feature to fix this. It would be great if  `sea_orm::DatabaseConnection` could be clonable in all cases.

File: examples/rocket_example/Cargo.toml [hunk 1]
```diff
@@ -23,7 +23,7 @@ serde_json = { version = "^1" }
 [dependencies.sea-orm]
 path = "../../" # remove this line in your own project
 version = "^0.5.0"
-features = ["macros", "runtime-tokio-native-tls"]
+features = ["macros", "mock", "runtime-tokio-native-tls"]
```
Comment (tyt2y3) [inline examples/rocket_example/Cargo.toml:26]: We dont actually have to add `mock` here right?

Comment (billy1624) [reply examples/rocket_example/Cargo.toml:26]: Not needed. I will revert it

Comment (billy1624) [reply examples/rocket_example/Cargo.toml]: Reverted

**Comment by tyt2y3** [comment]
I think we don't need this. Better keep it as is.

**Pull request closed without merging**